test/integration/utils/edge/config.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package edge
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+
+	"github.com/kubeedge/kubeedge/test/integration/utils/common"
+)
+
+//config.json decode struct
+type Config struct {
+	MqttEndpoint string `json:"mqttEndpoint"`
+	DeviceID     string `json:"deviceId"`
```

Comment:
as described in previous comment of script file , if we can avoid device id to be a part of configuration , we can get rid of it here as well.

Reply:
modified as suggested.